cdc/log.cc

```diff
@@ -1226,14 +1238,14 @@ struct process_change_visitor {
                 _enable_updating_state, nullptr, &_static_row_state, _clustering_row_states);
         visit_row_cells(v);
 
+        _builder.set_operation(log_ck, operation::update);
```

Comment:
What's this and how is this related to this commit?

Reply:
rebase error. removing.